src/screens/OrgList/OrgList.spec.tsx

```
@@ -52,20 +52,7 @@ vi.mock('shared-components/OrganizationCard/OrganizationCard', () => ({
-
-beforeEach(() => {
-  const ls = useLocalStorage();
-  setItem = ls.setItem;
-  removeItem = ls.removeItem;
-
-  // Seed guard keys for every test
-  setItem('IsLoggedIn', 'TRUE');
-  setItem('userId', '123'); // if this screen reads it
-  removeItem('AdminFor'); // must be absent (== undefined)
-});
+const { setItem, clearAllItems } = useLocalStorage();
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)